addons/l10n_eg_edi_eta/__manifest__.py

```diff
@@ -14,7 +14,6 @@
     'version': '0.2',
     'license': 'LGPL-3',
     'depends': ['account_edi', 'l10n_eg'],
-    'auto_install': ['l10n_eg'],
```

Comment:
From the name of the module, I have the feeling that it should be `auto_install: True`.
If you install account_edi later on and you have the eg loca, you probably also want the eg_edi stuff

Reply:
@Levizar It wasn't the case before and I don't want to change an existing behavior I don't know. That's why I make this PR: to go back to the previous behavior :)